stdlib/contextlib.pyi

```diff
@@ -40,7 +40,9 @@ _CM_EF = TypeVar("_CM_EF", bound=AbstractContextManager[Any] | _ExitFunc)
 @runtime_checkable
 class AbstractContextManager(Protocol[_T_co]):
     def __enter__(self) -> _T_co: ...
-    @abstractmethod
+    # While __exit__ is decorated with @abstractmethod in the runtime, it is also implemented.  Having the same decoration in the
```

Comment:
Whatever the outcome, we should make the same change to `AbstractAsyncContextManager.__aexit__`.